replace_split_find.py

```diff
@@ -0,0 +1,75 @@
+from my_exceptions import ValueOutOfRange
+
+# Без использования методов строк, напишите реализацию таких методов строк:
+# replace, split, find. Напишите функцию remove по индексу и по подстроке.
+
+
+def split_string(sentence='this is the string'):
+    split_values = []
+    tmp = ''
+    for c in sentence:
+        if c == ' ':
+            split_values.append(tmp)
+            tmp = ''
+        else:
+            tmp += c
+    if tmp:
+        split_values.append(tmp)
+    return split_values
+
+
+def replace_string(input_string='it is the test test test', sub_string='test', matches=2, new_sub_string='result'):
```

Comment:
Function will not work for
`
input_string = 'testhello'
sub_string = 'hello'
`